src/libraries/System.Net.Quic/src/System/Net/Quic/Internal/MsQuicSafeHandle.cs

```diff
@@ -88,7 +88,7 @@ internal sealed class MsQuicContextSafeHandle : MsQuicSafeHandle
     /// Holds a weak reference to the managed instance.
     /// It allows delegating MsQuic events to the managed object while it still can be collected and finalized.
     /// </summary>
-    private readonly GCHandle _context;
+    private GCHandle _context;
```

Comment:
This one should stay, otherwise the `.Free()` is executed on a defensive copy

https://github.com/dotnet/runtime/blob/1c7b419097c07a21e4a6bd8d7bde85942c1c015d/src/libraries/System.Net.Quic/src/System/Net/Quic/Internal/MsQuicSafeHandle.cs#L133

It probably is not a big deal since `ReleaseHandle` runs exactly once, but we still make sure to cleanup dangling pointers.